recipes/boost-ext-simd/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1.0)
+project(test_package)
+
+find_package(Boost.SIMD REQUIRED)
```

Until conan 2.0, `cmake` is our "toolchain" that applies some settings.
```suggestion
project(test_package)

include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
conan_basic_setup(TARGETS)

find_package(Boost.SIMD REQUIRED)
```